operator/duties/scheduler.go

```diff
@@ -414,18 +414,20 @@ func (s *Scheduler) ExecuteDuties(logger *zap.Logger, duties []*spectypes.Valida
 
 // ExecuteCommitteeDuties tries to execute the given committee duties
 func (s *Scheduler) ExecuteCommitteeDuties(logger *zap.Logger, duties committeeDutiesMap) {
-	for committeeID, duty := range duties {
-		committeeID := committeeID
-		duty := duty
-		logger := s.loggerWithCommitteeDutyContext(logger, committeeID, duty)
+	for _, committee := range duties {
+		duty := committee.duty
+		logger := s.loggerWithCommitteeDutyContext(logger, committee)
+		dutyEpoch := s.network.Beacon.EstimatedEpochAtSlot(duty.Slot)
+		logger.Debug("🔧 executing committee duty", fields.Duties(dutyEpoch, duty.ValidatorDuties))
```

Comment:
is this log necessary given theres the "starting duty processing" log [here](https://github.com/ssvlabs/ssv/blob/f969da086db0b9e33d6eb1ae7b8076dd0be1e1b8/protocol/v2/ssv/validator/committee.go#L207) which can receive the "duties" field also?

Reply:
discussed with @olegshmuelov and it would be simpler to rely on this log rather than passing `duties` down

Reply:
The purpose of this PR is to provide visibility at each step from execution to submission, specifically to track which duties are scheduled to be executed in a given slot. If an error occurs between the moment duties are sent for execution and the "starting duty processing" log, we wouldn't have any record of what duties were actually sent for execution. This log ensures we have that information available.